### PR TITLE
Added RO359B, board id 1022

### PR DESCRIPTION
**Pull request by ytsuboi**
CC @toyowata

### CODE REVIEW

File: test/info.py [hunk 1]
```diff
@@ -101,6 +102,7 @@
     (   0x1017,     'lpc11u35_hrm1017_if',                      None,               'mbed-HRM1017'                          ),
     (   0x1018,     'lpc11u35_ssci824_if',                      None,               'Switch-Science-mbed-LPC824'            ),
     (   0x1019,     'lpc11u35_sscity_if',                       None,               'Switch-Science-mbed-TY51822r3'         ),
+    (   0x1022,     'lpc11u35_ro359b_if',                       None,               'ro359b'                                ),
```
Comment (c1728p9) [inline test/info.py]: If the ro359b target can't be built using the online compiler you'll need to set it to None for now. If you do this just leave a TODO to the right with it.

Comment (ytsuboi) [inline test/info.py]: Hi @c1728p9 , is [this](https://github.com/mbedmicro/DAPLink/pull/266/commits/040e5a2b1141934a630b612ce325f37c8298a7be) right fix?

**Pull request merged**